.github/workflows/checkstyle.yml

```diff
@@ -0,0 +1,29 @@
+name: checkstyle
+
+on:
+    push:
+        branches:
+            - master
+            - 'dev/**'
```

Comment:
Shouldn't other branches such as `fix/**`, `feature/**`, `enhancement/**` etc. be included? Maybe it's something already discussed/approved but I'd rather ask.

Reply:
Alright 👍